sunpy/map/sources/hinode.py

```diff
@@ -98,6 +98,20 @@ def processing_level(self):
             return
         return int(lvl)
 
+    @property
+    def unit(self):
+        # XRT data values are in DN and are converted into DN/s if the data has been normalized.
+        # A tag starting with "XRT_RENORMALIZE" is added to the HISTORY tag in that case.
+        # Ref. Section 2.11 of the XRT Analysis Guide.
+        unit = self.meta.get('BUNIT')
```

Comment:
We do a bit more parsing of the unit information in the base class `unit` property.
```suggestion
        unit = super().unit
```

Reply:
ooh, I see